cmd/main.go

```diff
@@ -0,0 +1,128 @@
+package main
+
+import (
+	"embed"
+	"flag"
+	"io/fs"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+
+	"path/filepath"
+
+	"github.com/gin-contrib/static"
+	"github.com/gin-gonic/gin"
+	simulator "github.com/massalabs/thyra-playground-plugin/pkg"
+)
+
+func setupRouter(pluginPath string) *gin.Engine {
```

Comment:
I don't thing we are "duplicating the complexity". but maybe increasing our knowledge with a different framework.
I don't think we should stick to a technical choice which as been made before just because it was the good choice in a different context.
The good question is: does this framework suits our needs ?